main_he.tex

```diff
@@ -0,0 +1,61 @@
+% !TeX spellcheck = en_US
+\documentclass[12pt]{article}
+
+
+% Language setting
+\usepackage{fontspec} % For font settings
+\usepackage{polyglossia} % Better for handling multiple languages with XeLaTeX or LuaLaTeX
```

Comment:
Planning to get rid of it? Or is it required? I guess there will always be a tiny Latin component, like author, credits, right?

Reply:
Few of the names, also - numbers.